hypergan/samplers/random_walk_sampler.py

```diff
@@ -9,32 +9,38 @@ def __init__(self, gan, samples_per_row=8):
         self.y = None
         self.x = None
         self.step = 0
-        self.steps = 30
+        self.steps = 8
         self.target = None
 
     def _sample(self):
         gan = self.gan
-        z_t = gan.encoder.sample
```

Comment:
The difference between `uniform_encoder` and `encoder` is confusing, especially when we don't check that this is alphagan.